vtm/src/org/oscim/layers/tile/bitmap/BitmapTileLayer.java

```diff
@@ -129,7 +155,7 @@ private void setFade(MapPosition pos) {
             break;
         }
 
-        alpha = FastMath.clamp(alpha, 0f, mBitmapAlpha);
+        alpha = FastMath.clamp(alpha, 0f, 1f) * mBitmapAlpha;
```

Comment:
So instead of limit `alpha` on layer's value, we multiply that with it, not sure is the right approach?

Reply:
Why not?
If have a layer with default alpha, users don't want to change all FadeSteps to match with the default one. FadeStep should use full range from 0 to one independently of layers alpha.

Reply:
That change seems valid.